tutorials/community/reinforcement_learning/pipeline_reinforcement_learning_vertex_ai/pipeline_reinforcement_learning_vertex_ai.ipynb

```diff
@@ -914,70 +910,78 @@
     {
       "cell_type": "markdown",
-      "metadata": {
-        "id": "RelRBSFvibYc"
-      },
       "source": [
         "#### Build custom prediction container"
-      ]
+      ],
+      "metadata": {
+        "id": "RelRBSFvibYc"
+      }
     },
     {
       "cell_type": "code",
       "execution_count": null,
+      "source": [
+        "! gcloud builds submit --config $PREDICTION_CONTAINER_DIR/cloudbuild.yaml $PREDICTION_CONTAINER_DIR"
+      ],
+      "outputs": [],
       "metadata": {
         "id": "9uHbODeXibYd",
         "scrolled": true
-      },
-      "outputs": [],
-      "source": [
-        "! gcloud builds submit --config $PREDICTION_CONTAINER_DIR/cloudbuild.yaml $PREDICTION_CONTAINER_DIR"
-      ]
+      }
     },
     {
       "cell_type": "markdown",
-      "metadata": {
-        "id": "nW154IyqGilq"
-      },
       "source": [
         "## Author and run the RL pipeline\n",
         "\n",
         "You author the pipeline using custom KFP components built from the previous section, and [create a pipeline run](https://cloud.google.com/vertex-ai/docs/pipelines/run-pipeline#kubeflow-pipelines-sdk) using Vertex Pipelines. You can read more about whether to enable execution caching [here](https://cloud.google.com/vertex-ai/docs/pipelines/build-pipeline#caching). You can also specifically configure the worker pool spec for training if for instance you want to train at scale and/or at a higher speed; you can adjust the replica count, machine type, accelerator type and count, and many other specifications.\n",
         "\n",
         "Here, you build a \"startup\" pipeline that generates randomly sampled training data (with the Generator) as the first step. This pipeline runs only once."
-      ]
+      ],
+      "metadata": {
+        "id": "nW154IyqGilq"
+      }
     },
     {
       "cell_type": "code",
       "execution_count": null,
-      "metadata": {
-        "id": "cvXJzhSSGilq"
-      },
-      "outputs": [],
       "source": [
+        "from kfp.components import load_component_from_url\n",
+        "from google_cloud_pipeline_components import aiplatform as gcc_aip\n",
+        "\n",
+        "generate_op = load_component_from_url(\"https://raw.githubusercontent.com/Ark-kun/vertex-ai-samples/8233403b8269080a1a882ad9d4ae130ae888a08e/tutorials/community/reinforcement_learning/pipeline_reinforcement_learning_vertex_ai/src/generator/component.yaml\")\n",
+        "ingest_op = load_component_from_url(\"https://raw.githubusercontent.com/Ark-kun/vertex-ai-samples/8233403b8269080a1a882ad9d4ae130ae888a08e/tutorials/community/reinforcement_learning/pipeline_reinforcement_learning_vertex_ai/src/ingester/component.yaml\")\n",
+        "train_op = load_component_from_url(\"https://raw.githubusercontent.com/Ark-kun/vertex-ai-samples/4f2067fdb681763d95872e93a0a8cf6bae4eeb3d/tutorials/community/reinforcement_learning/pipeline_reinforcement_learning_vertex_ai/src/trainer/component.yaml\")\n",
+        "\n",
+        "\n",
```

Comment:
nit: Could you add instructions in the text cell above this about how you are loading these components? I think it would also be valuable to add a sentence to the 3 sections of the the Generator, Ingester and Trainer saying that we could run the component file to generate a YAML file, eg. "https://raw.githubusercontent.com/Ark-kun/vertex-ai-samples/......" and the YAML file would be used in the pipeline construction later on.